constants.js

```diff
@@ -0,0 +1,14 @@
+export const inventoryHeaderText = 'Inventory Management'
+export const getLocationName = (name) => `Location: ${name}`
+export const inventoryMenu = ['Inventory', 'Dispense']
+export const locationCookieName = "bahmni.user.location"
+export const headers = [
+  {
+    key: "productName",
+    header: "Product Name",
+  },
+  {
+    key: "actualQuantity",
+    header: "Actual Quantity",
```

Comment:
Should be called "Current Quantity"

Reply:
We went with the table header name given in the mock up. If required we can change this

Reply:
Actual Qty doesn't make sense. It should be either: `Available Quantity` or `Current Quantity` or `Quantity In Stock` or `Quantity in hand`.